galasa-managers-parent/galasa-managers-cicsts-parent/dev.galasa.cicsts.ceda.manager/src/main/java/dev/galasa/cicsts/ceda/internal/CedaManagerImpl.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed Materials - Property of IBM
+ * 
+ * (c) Copyright IBM Corp. 2020.
+ */
+package dev.galasa.cicsts.ceda.internal;
+
+import java.util.HashMap;
+import java.util.List;
+
+import javax.validation.constraints.NotNull;
+
+import org.osgi.service.component.annotations.Component;
+
+import dev.galasa.ManagerException;
+import dev.galasa.cicsts.CicstsManagerException;
+import dev.galasa.cicsts.ICeda;
+import dev.galasa.cicsts.ICicsRegion;
+import dev.galasa.cicsts.ceda.spi.ICedaManagerSpi;
+import dev.galasa.cicsts.spi.ICedaProvider;
+import dev.galasa.cicsts.spi.ICicstsManagerSpi;
+import dev.galasa.framework.spi.AbstractManager;
+import dev.galasa.framework.spi.IManager;
+
+@Component(service = { IManager.class })
+public class CedaManagerImpl extends AbstractManager implements ICedaManagerSpi,ICedaProvider {
+   
+	private ICicstsManagerSpi cicstsManager;
+	HashMap<ICicsRegion, ICeda> regionCeda = new HashMap<>();
+
```

Comment:
You still need the initialise method,   check that it is a Java test and if it is,   call "youAreRequired"